### PR TITLE
opkg-keyrings: Add metadata to make package visible on MAX

### PR DESCRIPTION
**Pull request by pratheekshasn**
### Summary of Changes

The package opkg-keyrings needs to become user visible on MAX so that the users can directly upgrade their keyrings from MAX through the dist feed.

### Justification

[#AB2866259](https://ni.visualstudio.com/DevCentral/_workitems/edit/2866259/)

### Testing

* [x] I have built the core package feed with this PR in place. (`bitbake packagefeed-ni-core`)
* [x] I have verified that the control file generated in the package has the metadata for UserVisible and DisplalyName under the rest of the info for opkg-keyrings.

### Procedure

* [x] I certify that the contents of this pull request complies with the [Developer Certificate of Origin](https://github.com/ni/nilrt/blob/HEAD/docs/CONTRIBUTING.md#developer-certificate-of-origin-dco).

### CODE REVIEW

File: recipes-devtools/opkg/opkg-keyrings_%.bbappend [hunk 1]
```diff
@@ -15,6 +15,8 @@ SRC_URI += " \
 	file://nilrt-feed-2023.gpg \
 "
 
+PACKAGE_ADD_METADATA_IPK:opkg-keyrings = "DisplayName: Opkg-Keyrings\nUserVisible: yes\n"
```
Comment (amstewart) [inline recipes-devtools/opkg/opkg-keyrings_%.bbappend]: Move this declaration below the task definition (`do_install`) below, per [the styleguide](https://github.com/ni/meta-nilrt/blob/nilrt/master/scarthgap/docs/styleguide.md#layout).

**Review by amstewart — CHANGES_REQUESTED** [review]
Change seems fine. Could you change the commit title to be like `opkg-keyrings: ...`, and add a proper description though?

File: recipes-devtools/opkg/opkg-keyrings_%.bbappend (hunk 1 above)
Comment (pratheekshasn) [reply recipes-devtools/opkg/opkg-keyrings_%.bbappend]: I have fixed this now.

**Comment by pratheekshasn** [comment]
> Change seems fine. Could you change the commit title to be like `opkg-keyrings: ...`, and add a proper description though?

I have fixed this now.

**Pull request closed without merging**